aws_iam_policies/aws_iam_policy_blocklist.yml

```diff
@@ -15,11 +15,11 @@ Severity: Medium
 Description: >
   This detects the usage of highly permissive IAM Policies that should only be assigned to a small number of users, roles, or groups.
 Runbook: >
-  https://docs.runpanther.io/alert-runbooks/built-in-policies/aws-iam-policy-blacklist-is-respected
+  https://docs.runpanther.io/alert-runbooks/built-in-policies/aws-iam-policy-blocklist-is-respected
```

Comment:
this url will have to be kept

Reply:
We could just delete that page from the docs and dupe it with the new title, that way the URL is changed